package.json

```diff
@@ -12,7 +12,7 @@
     "homepage": "https://github.com/plotly/dash-ag-grid",
     "main": "dash_ag_grid/dash_ag_grid.min.js",
     "scripts": {
-        "prepublishOnly": "rm -rf lib && babel src --out-dir lib --copy-files --config-file ./.babelrc && rm -rf lib/jl/ lib/*.jl",
+        "prepublishOnly": "rimraf -rf lib && babel src --out-dir lib --copy-files --config-file ./.babelrc && rimraf -rf lib/jl/ lib/*.jl",
```

Comment:
Looks like we have `rimraf` installed as a subdependency but that's fragile, let's list it explicitly in `devDependencies`.

```
~/plotly/dash-ag-grid> npm ls rimraf
dash-ag-grid@2.0.0 /Users/alex/plotly/dash-ag-grid
├─┬ eslint@8.39.0
│ └─┬ file-entry-cache@6.0.1
│   └─┬ flat-cache@3.0.4
│     └── rimraf@3.0.2
└─┬ react-docgen@5.4.3
  └─┬ estree-to-babel@3.2.1
    └─┬ c8@7.13.0
      └── rimraf@3.0.2 deduped

```